Failing test in org.loadui.testfx.TextInputControlsTest

## Issue

The following tests are failing:

```
org.loadui.testfx.TextInputControlsTest > shouldClearText FAILED
    java.lang.AssertionError at TextInputControlsTest.java:47
        Caused by: java.lang.AssertionError at TextInputControlsTest.java:47
```

As they do not fail on the Build-Server or are skipped (at least I couldn't find anything in the protocols). I assume the issue depends on my specific platform:
- Operating system: Apple OSX ElCapitan (10.11.4)
- Java: Oracle JDK8 (build 1.8.0_91-b14)
## Behaviour of test

``` Java
    @Test
    public void shouldClearText() {
        clickOn(TEXT_FIELD).write("Some text");
        verifyThat(TEXT_FIELD, hasText("Some text"));

        push(TAB); // To change focus from the TextField.

        clearTextIn(TEXT_FIELD);
        verifyThat(TEXT_FIELD, hasText(""));
    }
```

The text is correctly typed and verified in the first two lines. Within the rest of the test the last character is removed. And the test fails at the last verify.
## Analysis

The method clearTextIn() in in TextInputControls uses the following code to clear the TextField:

``` Java
fx.push(CONTROL, A).push(BACK_SPACE);
```

The keyboard shortcut to select all text on a Mac is 'command+A' actually.
## Proposed Solution

Either implement a platform specific behaviour or use the platform independent solution provided by `public static void clearTextIn(TextInputControl textInputControl)`

--- a/subprojects/testfx-legacy/src/main/java/org/loadui/testfx/controls/TextInputControls.java
+++ b/subprojects/testfx-legacy/src/main/java/org/loadui/testfx/controls/TextInputControls.java
@@ -25,7 +25,7 @@ import org.loadui.testfx.exceptions.NoNodesFoundException;
 
 import static javafx.scene.input.KeyCode.A;
 import static javafx.scene.input.KeyCode.BACK_SPACE;
-import static javafx.scene.input.KeyCode.CONTROL;
+import static javafx.scene.input.KeyCode.SHORTCUT;
 import static org.loadui.testfx.GuiTest.find;
 
 public class TextInputControls {
@@ -54,7 +54,7 @@ public class TextInputControls {
         };
 
         fx.clickOn(textControl);
-        fx.push(CONTROL, A).push(BACK_SPACE);
+        fx.push(SHORTCUT, A).push(BACK_SPACE);
     }
 
 }

